app/views/item_seasons/show.html.erb

```diff
@@ -1,13 +1,28 @@
-<h1 class="page__title"> <%= formatted_month_name(@item_season.month_index).capitalize %> </h1>
+<div class="content background-color-blue">
+  <h1 class="page__title banner-small m-bottom-m"> <%= formatted_month_name(@item_season.month_index).capitalize %> </h1>
 
-<table class="produce-table">
-  <% @item_season_produces.find_each do |produce| %>
-    <tr>
-        <td class="produce-table__name"><%= I18n.t("produce_items.names.#{produce.name}").capitalize %></td>
-        <td class="produce-table__category"><%= I18n.t("produce_items.categories.#{produce.category}") %></td>
-        <td class="produce-table__link_action"><%= link_to I18n.t('action.show'), produce, class: "link_action" %></td>
-    </tr>
-  <% end %>
-</table>
+  <div class="nav-actions-holder p-sides-m">
+    <h4><%= link_to I18n.t("action.back").upcase, item_seasons_path, class: "link_action back" %></h4>
+    <div class="nav-actions-months">
+      <%= link_to item_season_path(@item_season.month_index == 1 ? 12 : @item_season.month_index - 1) do %>
```

Comment:
**suggestion:** extract `@item_season.month_index == 1 ? 12 : @item_season.month_index - 1` to a helper with a meaningful name. I.e: `previous_month_index`